sqldb/sqlc/migrations/000002_invoices.up.sql

```diff
@@ -0,0 +1,57 @@
+-- The invoices table contains all the information shared by all the invoice 
+-- types. 
+CREATE TABLE IF NOT EXISTS invoices (
+        id INTEGER PRIMARY KEY,
+
+        memo TEXT,
+        amount_msat BIGINT NOT NULL, 
+        cltv_delta INTEGER NOT NULL,
+        expiry INTEGER NOT NULL,
+
+        features BLOB NOT NULL,
+
+        payment_addr BLOB UNIQUE,
+        preimage BLOB UNIQUE,
+        hash BLOB UNIQUE NOT NULL,
+        
+        state SMALLINT NOT NULL,
+        
+        payment_request TEXT UNIQUE, 
+
+        last_settled_index INTEGER,
+        last_settled_at TIMESTAMP,
+        last_amount_paid_msat BIGINT NOT NULL,
+
+        is_amp BOOLEAN NOT NULL,
+        is_hodl BOOLEAN NOT NULL,
```

Comment:
nit: hodl 😄

Reply:
haha I chekced it in our code and we call them [hodl invoices](https://github.com/lightningnetwork/lnd/blob/master/invoices/invoices.go#L421)

I think it's they real name [hold-invoices](https://bitcoinops.org/en/topics/hold-invoices/) though...